.gitignore

```diff
@@ -29,4 +29,5 @@ vendor/
 fastlane/report.xml
 google-services.json
 output-metadata.json
-service.properties
\ No newline at end of file
+service.properties
+wallet/schemas/*/*.json
```

Comment:
I don't think we should add this line because I believe that these files should be included to allow for database upgrades when the app is upgraded.

In this PR there is a `7.json` file included which seems to be leftover from working on the `feature-explore-dash` or some other story that is modifying the database.  In this PR the database is not included.